README.md

```diff
@@ -13,10 +13,45 @@ All query examples bellow are based on RavenDB's [sample data](https://ravendb.n
  
 ![Screen](https://github.com/ravendb/ravendb-grafana-datasource/raw/main/src/img/dashboard_screen.png)
 
-# Set RavenDB as your data source
-![SetDataSource](https://github.com/ravendb/ravendb-grafana-datasource/raw/main/src/img/SetDataSource.png)
+# Set RavenDB as your data source - Unsecure server
 
-[//]: # (![SetDataSource]&#40;../ravendb-grafana-datasource/img/SetDataSource.png&#41;)
+![unsecure settings](https://github.com/ravendb/ravendb-grafana-datasource/raw/main/src/img/unsecureSettings.png)
+
+1. Enter a name for this data source settings.
+2. Enter the database name from which to retrieve data.
+3. Enter your unsecure RavenDB server URL.
+4. Click Save & test to test and save this configuration.
+
+# Set RavenDB as your data source - Secure server
+
+![secure settings](https://github.com/ravendb/ravendb-grafana-datasource/raw/main/src/img/secureSettings.png)
+
+1. Enter a name for this data source settings.
+2. Enter the database name from which to retrieve data.
+3. Enter your secure RavenDB server URL.
+4. Toggle on TLS Client Auth.
+5. Enter the server name.  
+   The above example refers to a server in RavenDB Cloud. Replace 'danielle9897' with your own ID.  
```

Comment:
when user has paid version then he can choose domain and subdomain, so let's write that he needs to update hostname.

Reply:
Do you mean to write something like this:
```
The above example refers to a free server instance in RavenDB Cloud.
Replace it with your own hostname.
```
?

Reply:
done